src/toast/helpers/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Toast } from "../index";
+
+export const genId = (() => {
+  let count = 0;
+  return () => {
+    return (++count).toString();
+  };
+})();
+
+export const getToast = (toasts: Toast[], toastId: string): Toast => {
```

Comment:
export const getToast = (toasts: Toast[], toastId: string) => {


no need for explicit return type.